docs/DesignDocs/RemoteAttestationCollaterals.md

```diff
@@ -0,0 +1,285 @@
+Remote Attestation Collaterals
+=====
+
+Adding attestation APIs to support collaterals for remote attestation.  Collaterals are information from a second source that the user can provide for attestation verification.
+
+Motivation
+----------
+
+Currently, the existing collaterals used for Intel SGX quote verification are not exposed to the user.  Adding these new APIs allows the user to specify these collaterals instead of using the default collaterals used by OE SDK.
+
```

Comment:
it would make a stronger case to describe a actual use case for it. -- such as some scenarios have a need to validate a previously generate quote with collaterals collected when the quote was generated for auditing purpose, instead of using the most up-to-date collaterals.

Reply:
Good point! I added this other scenario.